### PR TITLE
ADD stock_analysis

### PR DESCRIPTION
**Pull request by eLBati**
This module allows you to analyse your current stock.
By default, it shows a graph where lines are products and columns are locations.
See screenshot

### CODE REVIEW

File: stock_analysis/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Stock Analysis
+==============
+
+This module allows you to analyse your current stock.
+By default, it shows a graph where lines are products and columns are locations.
```
Comment (elicoidal) [inline stock_analysis/README.rst]: s/graph/pivot table

File: stock_analysis/README.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Stock Analysis
+==============
+
+This module allows you to analyse your current stock.
```
Comment (elicoidal) [inline stock_analysis/README.rst]: Why only current ;)?=>
This module extends the stock reporting to allow better stock per location analysis.

File: stock_analysis/README.rst (hunk 1 above)
Comment (elicoidal) [inline stock_analysis/README.rst]: s/shows/adds

**Comment by eLBati** [comment]
@elicoidal thanks. Improved

**Comment by elicoidal** [comment]
LGTM

**Comment by tafaRU** [comment]
:+1:

File: stock_analysis/README.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Stock Analysis
+==============
+
+This module extends the stock reporting to allow better stock per location analysis.
+By default, it adds a pivot table where lines are products and columns are locations.
+See screenshot
```
Comment (rafaelbn) [inline stock_analysis/README.rst:11]: @eLBati where is the screenshot? Thanks!

Comment (eLBati) [inline stock_analysis/README.rst:11]: @rafaelbn see https://github.com/OCA/stock-logistics-reporting/pull/23, thanks

**Pull request merged**